tests/functional/cylc-install/00-simple.t

```diff
@@ -76,20 +75,6 @@ run_ok "${TEST_NAME}" cylc install --flow-name="${RND_WORKFLOW_NAME}" -C "${RND_
 contains_ok "${TEST_NAME}.stdout" <<__OUT__
 INSTALLED $RND_WORKFLOW_NAME/run1 from ${RND_WORKFLOW_SOURCE}
 __OUT__
```

Comment:
I'm not sure there is a test that the suite.rc deprecation message is printed on `cylc install`? If not, perhaps it should go here